cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/AbstractStageContext.java

```diff
@@ -99,7 +99,7 @@ public long getLogicalStartTime() {
     return pipelineRuntime.getLogicalStartTime();
   }
 
-  public int getMaxPreviewRecords() {
+  public int getMaxPreviewRecords(int maxPreviewRecordsLimit) {
```

Comment:
this won't compile right? Also please add `@Override` to method seems like it was missing previously.

Reply:
The StageContext class doesn't have this  method, should it be added?

Reply:
Sorry hadn't pushed that change